Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM python:3.5.1
+
+WORKDIR /app
+COPY . /app
+
+RUN pip install --upgrade --no-cache-dir -r app/requirements.txt \
+    && python setup.py install
```

Comment:
I think that's a problem with your machine; I have no trouble getting it to run, but it probably requires some config flags depending on your architecture.